resources/deployments/controller/controller_deployments_external_test.go

```diff
@@ -44,6 +46,16 @@ func makeDeviceAuthHeader(claim string) string {
 		base64.StdEncoding.EncodeToString([]byte(claim)))
 }
 
+func makeApi(router rest.App) *rest.Api {
+	api := rest.NewApi()
+	api.Use(
+		&requestlog.RequestLogMiddleware{},
```

Comment:
something I noticed on other services, will this throw lot of spam to stderr while executing tests? if so, maybe it would be good to redirect logs to `/dev/null`